metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java

```diff
@@ -0,0 +1,52 @@
+package eu.europeana.indexing.utils;
+
+import eu.europeana.indexing.tiers.model.MediaTier;
+import eu.europeana.indexing.tiers.model.MetadataTier;
+import eu.europeana.indexing.tiers.model.Tier;
+
+/**
+ * This enum lists the content and metadata tiers that exist.
+ */
+public enum RdfTier {
+
+  METADATA_TIER_0("http://www.europeana.eu/schemas/epf/metadataTier0", MetadataTier.T0),
```

Comment:
I believe the last part of the url should start with "#" and not a "/"?

Reply:
Good catch! However, I don't know. This document has them with slash:
https://docs.google.com/document/d/1BXdrIbyh2ufiC4WvDvisX8v1M89pAEVSne_tWXpHRdo/edit
And so does the description of this ticket:
https://europeana.atlassian.net/browse/MET-2008
However, the following ticket has it with hash (#):
https://europeana.atlassian.net/browse/MET-1987
Finally, this ticket has both versions (the table vs. the code snippet): 
https://europeana.atlassian.net/browse/MET-1988
So what is the right answer? I'm asking Valentine.

Reply:
As you can see in the ticket, the way I had it is correct.